app/javascript/components/create-board/create-board.tsx

```diff
@@ -85,19 +67,20 @@ export const CreateBoard: React.FC<CreateBoardProps> = ({ isCreateBoardOpen, set
     setCreateBoardOpen(false);
   };
 
-  const onEmojiClick = (name: string) => (event: React.MouseEvent, emojiObject: IEmojiData) => {
-    setformData({
-      ...formData,
-      [name]: emojiObject
-    });
-    setEmojiData({
-      emojiColumnName: '',
-      emojiPickerOpen: false
-    });
-  };
+  const onEmojiClick =
+    (name: string) => (event: MouseEvent, emojiObject: IEmojiData) => {
+      setformData({
+        ...formData,
+        [name]: emojiObject,
+      });
+      setEmojiData({
+        emojiColumnName: "",
+        emojiPickerOpen: false,
+      });
+    };
 
   return (
-    <div className={isCreateBoardOpen ? 'modal-visible' : 'modal-hidden'}>
+    <div className={isCreateBoardOpen ? "modal-visible" : "modal-hidden"}>
       <div className={style.formWrapper}>
         {emojiData.emojiPickerOpen && (
           <div className={style.emojiPickerWrapper}>
```

Comment:
Type '(event: MouseEvent, emojiObject: IEmojiData) => void' is not assignable to type
 '(event: MouseEvent<Element, MouseEvent>, data: IEmojiData) => void'.